2-WebApp-graph-user/2-1-Call-MSGraph/Startup.cs

```diff
@@ -40,6 +41,32 @@ public void ConfigureServices(IServiceCollection services)
                .AddMsal(Configuration, new string[] { Constants.ScopeUserRead })
                .AddInMemoryTokenCaches();
 
+            /*
+               // or use a distributed Token Cache by adding 
+                           .AddDistributedTokenCaches();
+
+               // and then choose your implementation. 
+               // See https://docs.microsoft.com/en-us/aspnet/core/performance/caching/distributed?view=aspnetcore-2.2#distributed-memory-cache
+
+               // For instance the distributed in memory cache (not cleaned when you stop the app)
```

Comment:
Prefer to use the term `clear` instead of `clean`, i.e. (not cleared when you stop the app)

Reply:
Done, Thanks, @bgavrilMS